lib/index.js

```diff
@@ -346,17 +346,11 @@ export async function bundle(options = {}) {
                   require.resolve('@babel/preset-env'),
                   {
                     useBuiltIns: 'entry',
-                    corejs: 3,
+                    corejs: '3.21',
+                    bugfixes: true,
                     debug: command.webpackBabelDebug === 'true'
```

Comment:
I recommend we make it our default behavior not to support IE11 now. This removes the regenerate polyfill which can create massive bloat for very little to no gain in terms of user base.

```suggestion
                    debug: command.webpackBabelDebug === 'true',
                    targets: ['defaults', 'not IE 11'],
```